src/commonReducers/clients/clientSlice.js

```diff
@@ -0,0 +1,46 @@
+import { createAsyncThunk, createSlice } from '@reduxjs/toolkit';
+import sources from 'api/sources';
+
+
+export const fetchClientsById = createAsyncThunk(
```

Comment:
I really think this should be called `fetchClients` by userId is implied imo. 

and mostly because fetchClientsById sounds like it means you would provide the client ID.